src/test/resources/personVT1.json

```diff
@@ -18,7 +18,7 @@
     },
     {
         "title": "Name in passport",
-        "function": "#/functionDefinitions/concat($lastName, $firstName)",
+        "function": "#/functionDefinitions/userDefinedConcat($lastName, $firstName)",
```

Comment:
Create a sample with both in-line and provider and ref for test purposes.

Reply:
done